relay-profiling/src/android.rs

```diff
@@ -203,11 +207,14 @@ fn parse_profile(payload: &[u8]) -> Result<AndroidProfilingEvent, ProfileError>
 
 pub fn parse_android_profile(
     payload: &[u8],
+    client_sdk: Option<ClientSdk>,
     transaction_metadata: BTreeMap<String, String>,
     transaction_tags: BTreeMap<String, String>,
 ) -> Result<Vec<u8>, ProfileError> {
     let mut profile = parse_profile(payload)?;
 
+    profile.metadata.client_sdk = client_sdk;
```

Comment:
Should we handle this the same way we handle `dist` and `release` and use `transaction_metadata` to pass the data along?